generators/server/templates/_pom.xml

```diff
@@ -941,9 +963,6 @@
                     Enable the line below to have remote debugging of your application on port 5005
                     <jvmArguments>-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005</jvmArguments>
                     -->
-                    <arguments>
-                        <argument>--spring.profiles.active=dev</argument>
```

Comment:
Not when running via Maven/Gradle -- because in those cases the value in application.yml will have been filled in. What we discussed (I thought) was only for 

- overriding the default on an existing war file
- running in IDE (when the unsubstituted value will be read)

I've confirmed these cases to work. Sorry if I misunderstood anything?